src/modules/ROOT/pages/changelog/202209/02.json

```diff
@@ -0,0 +1,393 @@
+      ]
+    },
+    {
+      "what": [
+        "Lower and upper case character inputs in the password constraints section were changed from default to number.",
+        "Also, the labels of the two fields were reworded to match the style of the other inputs in the password constraints section."
+      ],
+      "why": [
+        "The input type was changed to avoid sending strings to the database, the rewording was done for overall consistency"
+      ],
+      "how": [
+        "The translations in corteza-locale were changed, input type was set to number"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-locale/commit/703516c9e23bf5373de01ba073b11426cbe66390", 
+        "https://github.com/cortezaproject/corteza-webapp-admin/commit/aad919f90700a5b8de8f15c299ddf351f3c2e1b2"
+      ]
+    },
+    {
+      "what": [
+        "The return type of the password constraint fields and a bit of rewording for upper and lowercase constraints."
+      ],
+      "why": [
+        "Password constraints are numbers and should be saved as such and not strings.",
+        "Rewording was done to match the consistency of the other password constraints inputs"
+      ],
+      "how": [
+        "'.number' was appended to the password constraint's input model, translation strings' context was changed"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-locale/commit/703516c9e23bf5373de01ba073b11426cbe66390",
+        "https://github.com/cortezaproject/corteza-webapp-admin/commit/a2480009e6f1c59c10569343bef98270ce0968d2"
+      ]
+    },
+    {
+      "what": [
+        "Documentation about sanitizers and fixed the example links in module field validation tab to now point to the proper anchors."
+      ],
+      "why": [
+        "Documentation was lacking information about field value sanitization."
+      ],
+      "how": [
+        "The documentation was updated with a new important section describing this behavior.",
+        "And documentation links in module field validators tab now point to validators/sanitizers respectively"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-webapp-compose/commit/9af28f269ae4752c036522273ec9c1af6a14efd2"
+      ]
+    }
+  ],
+
+  "fixed": [
+    {
+      "what": [
+        "Incorrect user count from the system metrics report due to system-defined users"
+      ],
+      "how": [
+        "System users are now excluded from the metric report, we didn't exclude them from the total, but they were excluded from the per-day aggregation"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-server/commit/fd6465d0f95d78210401f0d0be9c16e1290341af"
+      ]
+    },
+    {
+      "what": [
+        "The timestamp in the details of an action log being different as the one in search results"
+      ],
+      "how": [
+        "The custom format function IocFullDateTime was applied to the timestamp"
```

Comment:
You can skip low-level implementation details; something like `applying a custom formatting to the timestamp`